python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -142,6 +142,25 @@ def __set_elem__(tensor, i, e):
     return gradient_flat.reshape(tensor_to_check.shape())
 
 
+def skip_check_grad_ci(func):
+    """Decorator to skip check_grd CI.
+       
+       Check_grad is required for Op test cases. However, there are specical cases 
+       that do not need to check_grad. The decorator is used to avoid failures of 
```

Comment:
This decorator is used to skip the check_grad of the above cases.

Reply:
Done.